src/lib/interactionManager/credentialOfferFlow.ts

```diff
@@ -77,13 +78,26 @@ export class CredentialOfferFlow extends Flow<
       }
     })
 
-    const validArr = (this.state.credentialsValidity = await JolocomLib.util.validateDigestables(
+    const validArr = (this.state.credentialsValidity = await this.validateCredentials(
       signedCredentials,
     ))
+
     this.state.credentialsAllValid = validArr.every(v => v)
     return true
   }
 
+  private async validateCredentials(signedCredentials: SignedCredential[]) {
+    return Promise.all(
+      signedCredentials.map(async cred => {
+        const validDigestable = await JolocomLib.util.validateDigestable(cred)
+        const validIssuer = cred.issuer !== this.ctx.participants.requester.did
```

Comment:
Ummm no self-signed credentials??

Reply:
Should you be able to receive self-signed credentials from a different identity?